### PR TITLE
test: pytest: async test functions should use get_running_loop()

### PR DESCRIPTION
**Pull request by jelly**

### CODE REVIEW

**Comment by jelly** [comment]
Bah:

```
test/pytest/test_peer.py:169
  test/pytest/test_peer.py:169: PytestDeprecationWarning: test_await_cancellable_connect_close is asynchronous and explicitly requests the "event_loop" fixture. Asynchronous fixtures and test functions should use "asyncio.get_running_loop()" instead.
    @pytest.mark.asyncio
```

We can merge it or you can request a fixup

File: test/pytest/test_bridge.py [hunk 1]
```diff
@@ -689,7 +689,8 @@ async def recv_one():
 
 
 @pytest.mark.asyncio
-async def test_large_upload(event_loop: asyncio.AbstractEventLoop, transport: MockTransport, tmp_path: Path) -> None:
+async def test_large_upload(transport: MockTransport, tmp_path: Path) -> None:
+    event_loop = asyncio.get_running_loop()
```
Comment (allisonkarlitskaya) [inline test/pytest/test_bridge.py:693]: Pretty sure we don't have this on 3.6?

Comment (jelly) [reply test/pytest/test_bridge.py:693]: Argh, good point, but tests pass? Don't we run tox on 3.6 here?

Comment (jelly) [reply test/pytest/test_bridge.py:693]: Checked docs, introduced in 3.7

Comment (allisonkarlitskaya) [reply test/pytest/test_bridge.py:693]: I think we might be polyfilling this somewhere.  Seems a bit weird to depend on it, though...

Comment (jelly) [reply test/pytest/test_bridge.py:693]: Agreed, but can we ignore deprecations with "required for 3.6"?

Comment (jelly) [reply test/pytest/test_bridge.py:693]: Agreed, this is weird. Dropping this PR.

**Pull request closed without merging**